tests/integration_tests/test_hyperopt_ray_horovod.py

```diff
@@ -273,8 +276,8 @@ def test_hyperopt_run_hyperopt(csv_filename, ray_mock_dir, ray_cluster_7cpu):
                 "lower": 0.001,
                 "upper": 0.1,
             },
-            output_feature_name + ".output_size": {"space": "randint", "lower": 2, "upper": 32},
-            output_feature_name + ".num_fc_layers": {"space": "randint", "lower": 2, "upper": 6},
+            output_feature_name + ".output_size": {"space": "randint", "lower": 2, "upper": 8},
```

Comment:
`test_hyperopt_run_hyperopt` is a good one to test this feature out with. You can set `max_concurrent_trials=auto`, the bump up the `cpus_per_trial=2`. If your code works, then the test should pass, but if it doesn't, it will hang. So we can test this logic for free without needing to add a new test.

Reply:
@tgaddair So that test is currently skipped because of some sort of issues with dynamic resource allocation.

Is it worth adding to `test_hyperopt_executor`, specifically the test with variant generator?